frontend/public/components/catalog/catalog-page.jsx

```diff
@@ -54,7 +59,32 @@ export class CatalogListPage extends React.Component {
       imageStreamsItems = this.normalizeImagestreams(imagestreams.data, 'ImageStream');
     }
 
-    return _.sortBy([...clusterServiceClassItems, ...imageStreamsItems], 'tileName');
+    if (clusterServiceVersions) {
+      const imgFor = (desc) => _.get(desc.csv, 'spec.icon')
+        ? `data:${_.get(desc.csv, 'spec.icon', [])[0].mediatype};base64,${_.get(desc.csv, 'spec.icon', [])[0].base64data}`
```

Comment:
Might be worth creating a utility for this since we're doing it several places now.